guides/source/6_1_release_notes.md

```diff
@@ -257,7 +257,7 @@ Please refer to the [Changelog][active-record] for detailed changes.
 
 *   Named scope chain does no longer leak scope to class level querying methods.
 
-        class class User < ActiveRecord::Base
+        class User < ActiveRecord::Base
```

Comment:
@edudepetris While you're there, what would you think about updating the parent class to `ApplicationRecord` from `ActiveRecord::Base`, to be consistent with what is generated since Rails 5?

Reply:
@andyatkinson https://github.com/rails/rails/pull/40601